genai-perf/genai_perf/utils.py

```diff
@@ -97,8 +97,8 @@ def load_json_str(json_str: str, func: Callable = lambda x: x) -> Dict[str, Any]
           run validation checks on the object. Defaults to identity function.
     """
     try:
-        return func(json.loads(json_str))
-    except json.JSONDecodeError:
+        return func(orjson.loads(json_str))
```

Comment:
As far as I know, from a quick search, orjson doesn't strictly parses json the same exact way as the stock Python implementation, especially regarding utf-8 sequences. Maybe add a comment up to indicate this detail, and to refer the reader to orjson's documentation for the specificities of its parsing?

Reply:
Good point. Added a comment!